File: app/src/main/java/toru/io/my/viperusage/model/InstagramResponse.java
package toru.io.my.viperusage.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by toruchoi on 08/10/2017.
 */

public class InstagramResponse {
    public List<InstagramItemModel> items;

    @SerializedName("more_available")
    public boolean moreAvailable;

    public String status;
}

File: kotlin_dagger_app/src/main/java/my/liewjuntung/kotlindaggerapp/model/response/InstagramResponse.kt
package my.liewjuntung.kotlindaggerapp.model.response

import com.google.gson.annotations.SerializedName

/**
 * Created by toruchoi on 24/10/2017.
 */
data class InstagramResponse(val items:List<InstagramItemModel>, @SerializedName("more_available") val more:Boolean, val status:String)